core/cap-0021.md

```diff
@@ -297,29 +298,33 @@ fail the second validation, for instance if a `BUMP_SEQUENCE`
 operation made the sequence number invalid.  Whenever a transaction
 fails validation during execution, the `sourceAccount` loses the fee.
 
-Because `PreconditionsV2` specifies multiple pre-conditions, there may
-be multiple reasons why a transaction is invalid.  If any of the
-reasons is permanent---namely the `maxTime` or `maxLedger` has been
-exceeded--then the transaction is rejected with
-`TransactionResultCode` `txTOO_LATE`.  Otherwise, if all the
-unsatisfied preconditions are based on values that could later be
-valid, then the transaction is rejected with `txTOO_EARLY`.  If
-neither of the above conditions holds (`txTOO_EARLY` and `txTOO_LATE`
-do not apply), but one of the `extraSigners` is unsatisfied, then the
-transaction fails with `txBAD_AUTH`.
+Because `PreconditionsV2` specifies multiple pre-conditions, there may be
```

Comment:
I don't think we specify this, but I assume `txBAD_SEQ` takes precedence over any of these failure conditions. Is that correct?

Reply:
`txBAD_SEQ` is before `txBAD_MIN_SEQ_AGE_OR_GAP` and `txBAD_AUTH` but after the others.